install/kubernetes/helm/istio/values.yaml

```diff
@@ -358,7 +358,16 @@ global:
   #   services or ServiceEntries for the destination port
   outboundTrafficPolicy:
     mode: REGISTRY_ONLY
-  
+
+  # configRootNamespace: istio-config
+
+  # set the default visibility of a service, service entry, virtual service, destination
+  # rule across one or more namespaces.
+  # * implies these objects are visible to all namespaces, enabling any sidecar to talk to any other sidecar.
+  # . implies these objects are visible to only the current namespace where the sidecar is running
+  # ~ implies these objects are not visible to anyone by default. Sidecars will bootup with no outbound configs.
+  defaultConfigExportSetting: '*'
+
```

Comment:
See above, should be a list - maybe default is . and istio-telemetry ( so prometheus can pull stuff ), or some other ns that needs to see all services.

Reply:
backward compatibility.. Also, we currenly only support . and *.

Reply:
But the API is not based on what we currently support, we can't churn the API every time we implement a new feature. 

We can document that in 1.1 we have a limitation, and not change the API in 1.2

Reply:
done